gradle-plugin/src/main/kotlin/com/github/buildeye/utils/FileSystemUtils.kt

```diff
@@ -0,0 +1,10 @@
+package com.github.buildeye.utils
+
+import org.gradle.internal.impldep.org.apache.commons.codec.digest.DigestUtils
```

Comment:
Please try not to use Gradle internal API, and especially don't use Gradle internal dependencies (in this case you're referencing `apache-commons` relocated to `org.gradle.internal.impldep` package). If you absolutely need to use a library, please add a compile dependency on that library. Since Gradle does not provide runtime class isolation for its plugins, you need to relocate such dependencies to `com.github.buildeye` package (via Shadow Gradle plugin https://github.com/johnrengelman/shadow).
Otherwise your code will break when Gradle updates or removes an internal dependency.